static/src/javascripts/projects/common/modules/experiments/tests/play-video-on-fronts.js

```diff
@@ -20,17 +27,58 @@ define([
 
         this.canRun = function () {
             // Only videos that are links to video pages have data-embed-paths
-            return config.page.isFront && $('.fc-item--has-video-main-media .js-video-play-button').length > 0;
+            // Or article pages that have been linked to from test-enhanced cards
+            return (config.page.isFront && $('.fc-item--has-video-main-media .js-video-play-button').length > 0) ||
+                (config.page.contentType === 'Article' && hasAutoplayHash);
```

Comment:
Adding a second page to the canrun function is going to make this a bit more difficult to analyse. We'd have to include `and publication_date is null` to the query to try to get only fronts. This isn't too bad, but does mean this test won't be analysable in the standard way. Ideally this is moved outside of canrun. @desbo 

I'd go ahead with it, but we'll have to ignore the Ophan pages when looking at sample size